examples/mnist/Instructions.md

```diff
@@ -0,0 +1,27 @@
+<p align="center"><b>Setup, build and run tiny-ddn on various OS</b><p align="center">
+
+<b>tiny-dnn</b> is a C++11 implementation of deep learning. It is suitable for deep learning on limited computational resource, embedded systems and IoT devices.
+
+
+<b>How to Install/Build Tiny-dnn On Mac/Linux</b>  
+<b>Step:1</b> Install Cmake from https://cmake.org/install/  
+<b>Step:2</b> Open Terminal (Press CMD+SpaceBar Enter Terminal)    
+<b>Step:3</b>
+
+1. cd ~  
+2. git clone https://github.com/tiny-dnn/tiny-dnn.git  
+3. cd tiny-dnn  
+4. mkdir build  
+5. cd build  
+6. cmake .. -DBUILD_EXAMPLES=ON  
+7. make  
+8. cd examples/  
+9. ./example_mnist_train ~/tiny-dnn/data/  
```

Comment:
Oh no, you are right. hmm... so after step 8, current directory is `build/examples`.